README.md

```diff
@@ -1,6 +1,6 @@
-# Node.js Long-term Support Working Group
+# Node.js Release and Long-term Support Working Group
 
-# LTS schedule<sup>1</sup>
+## LTS schedule<sup>1</sup>
```

Comment:
Since this table contains non-LTS releases, though not fully documented, how about _Release Schedule_?

Or perhaps at this point a general _Release_ table would be in order since this WG would now be responsible for all releases?